### PR TITLE
Code for hearing date - copied from old workallocation2 folder

### PR DESCRIPTION
**Pull request by Surekha-HMCTS**
### Change description ###

Code for hearing date - copied from old workallocation2 folder 

Referred: https://github.com/hmcts/rpx-xui-webapp/pull/2126/files#diff-fe202ecd0b728d60b77117583fd787e43454ca90a74bdf57703934b990c76d48

### CODE REVIEW

File: api/.env.defaults [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NODE_CONFIG_DIR=../config
 ALLOW_CONFIG_MUTATIONS=1
-NODE_CONFIG_ENV=development
+NODE_CONFIG_ENV=demo
```
Comment (DavidJayakumar) [inline api/.env.defaults:3]: Revert this

**Review by DavidJayakumar — APPROVED** [review]
Loods good

**Review by connorpgpmcelroy — COMMENTED** [review]
Just have one question

File: api/workAllocation/util.ts [hunk 2]
```diff
@@ -99,6 +99,14 @@ export function prepareCaseWorkerForLocationAndService(baseUrl: string, location
...
   // Assign actions to the tasks on the data from the API.
   if (req.body && req.body.searchRequest && req.body.searchRequest.pagination_parameters) {
+    const sortingParameters = req.body.searchRequest.sorting_parameters;
+    if (sortingParameters && sortingParameters.length > 0) {
+      sortingParameters.forEach( sortParam => {
+        if (sortParam.sort_by === 'hearing_date') {
+          sortParam.sort_by = 'caseName';
+        }
+      });
+    }
```
[the remark this answers is not in the document]
Comment (Surekha-HMCTS) [reply api/workAllocation/util.ts:109]: yes, it is a temporary code to sort the tasks as per case name

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**